packages/manager/apps/dedicated/client/app/dedicated/server/details/translations/Messages_en_GB.json

```diff
@@ -1272,12 +1272,12 @@
   "server_error_bandwidth_order": "An error has occurred ordering bandwidth: {{error}}",
   "server_configuration_task_INFRA_002_VIRTUAL_NETWORK_INTERFACE_GROUP": "Group network interfaces",
   "server_configuration_task_INFRA_002_VIRTUAL_NETWORK_INTERFACE_UNGROUP": "Detach network interfaces",
-  "server_configuration_no_intervention": "On-site interventions",
+  "server_configuration_no_intervention": "Disable on-site interventions",
   "server_configuration_on": "On",
   "server_configuration_off": "Off",
-  "server_configuration_no_intervention_activate_success": "On-site intervention is enabled on server {{serverName}}",
+  "server_configuration_no_intervention_activate_success": "On-site intervention is disabled on server {{serverName}}",
```

Comment:
@radireddy, i think there is a mistake, the key indicate **...activate_success** and the content indicate "... disabled ...".
There are raisons for that ?